CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# [0.4.2](https://github.com/mattrglobal/pairing_crypto/compare/0.4.2...0.4.3) (2024-10-21)
+
+React native >=72 support, gradle 8 support
```

Comment:
Should we at the supported RN upper version too?

Reply:
updated